ui/src/pages/ShoeMap.tsx

```diff
@@ -1,14 +1,45 @@
 import * as React from "react";
 import styled from "styled-components";
-import { Map, TileLayer, ZoomControl } from "react-leaflet";
+import { AttributionControl, Map, TileLayer, ZoomControl } from "react-leaflet";
+import Control from "react-leaflet-control";
 import "leaflet/dist/leaflet.css";
 import { PinCluster } from "../components/PinCluster";
 import { StoryDrawer, StoryDrawerState } from "../components/StoryDrawer";
 import { useState } from "react";
+import { colors } from "../styles";
 
 const StyledMap = styled(Map)`
   height: 100vh;
   width: 100vw;
+
+  .leaflet-control {
+    border: none;
+    box-shadow: 0px 0px 25px rgba(0, 0, 0, 0.1);
+    margin: 0px 24px 24px 0px;
+  }
+`;
+
+const StyledHelpIcon = styled.button`
+  height: 30px;
+  width: 30px;
+  font-size: 20px;
+  font-weight: 500;
+  border: none;
+  background-color: ${colors.white};
+  border-radius: 5px;
+  box-shadow: 0px 0px 25px 5px rgba(0, 0, 0, 0.1);
+
```

Comment:
For some reason the help button looks weird for me unless I add:
`  text-align: center;
  padding-bottom: 8px;
  min-width: 30px;`
Otherwise the "?" is not centered in the button and the button is not the same width as the zoom control.

Reply:
Yeah I think the + and - are slightly off-center too 
![image](https://user-images.githubusercontent.com/19617248/99916592-06e30a00-2cd9-11eb-9b42-fe6a114511fe.png)
More space at the top?

Reply:
@m-penny Hmm that's weird, do you mind attaching a screenshot?